id.html

```diff
@@ -171,13 +171,13 @@ <h4>
             </i>Visual
           </h4>
         </a>
-        <p>Memodifikasi animasi karakter, memodifikasi ekspresi karakter, menampilkan item dari Peta Interaktif Teyvat, buat browser Unity, Pengelihatan tak terbatas, memodifikasi UID, memodifikasi tampilan profile card, kustomisasi cuaca, membuka batasan FPS, free camera, memodifikasi tekstur, Packet Grabber, dan Kustomisasi Private Key.
+        <p>Dapat mengubah animasi karakter, mengubah ekspresi karakter, menampilkan item dari Map Interactive, memunculkan browser Unity, pengelihatan tak terbatas, mengubah UID (hanya visual), mengubah tampilan Profile Card, kustomisasi cuaca, membuka batasan FPS, free camera, mengubah tekstur, Packet Grabber, dan kustomisasi Private Key.
```

Comment:
`Map interactive` is better as `Map interaktif` for consistency
`Free camera` can be translated as `kustomisasi kamera` or `kamera kustom` pick whichever fits the context and consistency.